test/FunctionalTests/Server/UnaryMethodTests.cs

```diff
@@ -346,7 +346,7 @@ public async Task SingletonService_PrivateFieldsPreservedBetweenCalls()
 
             // Act 1
             var response = await Fixture.Client.PostAsync(
-                "SingletonCount.Counter/IncrementCount",
+                "singleton_count.Counter/IncrementCount",
```

Comment:
I do find a mix of snake_case and PascalCase looks odd though. How do other languages that have different casing schemes handle this?

Reply:
I think package names should be  `lower_case_with_underscores` according to the .proto style guide.

Because `singleton_count.Counter/IncrementCount` is part of the method URL, it needs to contain a verbatim protobuf package, so that the URL is the same across languages. Note that under normal use, both client and server will generate this URL internally for you so you won't see it at all.